src/gtimelog/main.py

```diff
@@ -1786,9 +1786,9 @@ def __init__(self):
         self.connect('notify::recipient', self.update_already_sent_indication)
         self.bind_property('body', self.get_buffer(), 'text',
                            GObject.BindingFlags.BIDIRECTIONAL)
-        # GTK+ themes other than Adwaita ignore the 'monospace' property and
-        # use a proportional font for text widgets.
-        self.override_font(Pango.FontDescription.from_string("Monospace"))
+        css_provider = Gtk.CssProvider()
+        css_provider.load_from_data("textview {font-family: monospace;}")
+        Gtk.StyleContext.add_provider_for_screen(self.get_screen(), css_provider, Gtk.STYLE_PROVIDER_PRIORITY_USER)
```

Comment:
I wonder if it would make sense to add this to src/gtimelog/gtimelog.css instead, which is already hooked up in Application.do_startup() (but uses Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION).

Also., I'd prefer not to lose the comment mentioning that this is a workaround for [certain alternative GTK themes](https://github.com/gtimelog/gtimelog/issues/81#issuecomment-170923964).  It could go as a comment in the CSS file, I suppose.

Reply:
Hm, also wouldn't this style rule apply to both textviews (main and reporting)?  Only the report textview is supposed to be monospace.

Reply:
Ups, I wasn't aware of the existing CSS file. I amended the commit and moved the settings (and comment) to the existing CSS file.